presidio-image-redactor/tests/conftest.py

```diff
@@ -45,3 +50,26 @@ def get_image_recognizerresult():
 @pytest.fixture(scope="module")
 def image_analyzer_engine():
     return ImageAnalyzerEngine()
+
+
+@pytest.fixture(scope="module")
+def mock_instance():
```

Comment:
nit: `mock_instance` is a very generic name, and this package does different things. Consider changing to `get_mock_dicom_instance` or something similar. We can also address this in a future PR.